File: Juan/P3/Ruby/Napakalaki/pkg/Napakalaki-0.0.1/lib/CombatResult.rb
#encoding: utf-8

#Versión 3.0
module NapakalakiGame
module CombatResult
    
    WINGAME = :WINGAME
    WIN = :WIN
    LOSE = :LOSE
 
    def to_s
      if [CombatResult::WINGAME]          
          return "Has ganado el juego"
      end
                
          if [CombatResult::WIN]
             return "Has ganado"
          end
          
      if [CombatResult::LOSE]
          return "Has perdido"
      end
     
      
    end
end
end

File: Juan/P4/Java/Napakalaki/src/NapakalakiGame/CombatResult.java
package NapakalakiGame;

/**
 * @version 3.1
 * @author juane
 */
public enum CombatResult {
    WINGAME,WIN,LOSE;
    
    @Override
    public String toString(){
        
        if(this.equals(WIN)){
            return "Ganas";
        }
        else if (this.equals(LOSE)){
            return "Has perdido ohh";
        }
        
        else if(this.equals(WINGAME)){
            return "Enhorabuena has ganado";
        }
        else
            return "HA OCURRIDO UN ERROR";
    }
}
